ui/admin/tests/acceptance/auth-methods/selectors.js

```diff
@@ -0,0 +1,92 @@
+export const FIELD_IDP_CERTS_ADD_BTN = '[name=idp_ca_certs] button';
+export const FIELD_MAX_AGE = '[name=max_age]';
+export const FIELD_MAX_AGE_VALUE = '5';
+export const FIELD_API_URL_PREFIX = '[name=api_url_prefix]';
+export const FIELD_API_URL_PREFIX_VALUE = 'Api url prefix';
+export const FIELD_PROMPTS = '[name=prompts]';
+export const FIELD_PROMPTS_CONSENT = '[id=consent]';
+export const FIELD_PROMPTS_SELECT_ACCOUNT = '[id=select_account]';
+export const NEW_DROPDOWN = '[data-test-new-dropdown] button';
+export const FIELD_URLS = '[name=urls]';
+export const FIELD_URLS_VALUE = 'url1,url2';
+export const FIELD_CERTIFICATES = '[name=certificates] textarea';
+export const FIELD_CERTIFICATES_VALUE = 'certificate';
+export const FIELD_CERTIFICATES_ADD_BTN = '[name=certificates] button';
+export const FIELD_CLIENT_CERTIFICATE = '[name=client_certificate]';
+export const FIELD_CLIENT_CERTIFICATE_VALUE = 'Client certificate value';
+export const FIELD_CLIENT_CERTIFICATE_KEY = '[name=client_certificate_key]';
+export const FIELD_CLIENT_CERTIFICATE_KEY_VALUE =
+  'Client certificate key value';
+export const FIELD_START_TLS = '[name=start_tls]';
+export const FIELD_INSECURE_TLS = '[name=insecure_tls]';
+export const FIELD_BIND_DN = '[name=bind_dn]';
+export const FIELD_BIND_DN_VALUE = 'Bind dn';
+export const FIELD_BIND_PASSWORD = '[name=bind_password]';
+export const FIELD_BIND_PASSWORD_VALUE = 'Bind password value';
+export const FIELD_UPN_DOMAIN = '[name=upn_domain]';
+export const FIELD_UPN_DOMAIN_VALUE = 'Upn domain value';
+export const FIELD_DISCOVER_DN = '[name=discover_dn]';
+export const FIELD_ANON_GROUP_SEARCH = '[name=anon_group_search]';
+export const FIELD_USER_DN = '[name=user_dn]';
+export const FIELD_USER_DN_VALUE = 'User dn value';
+export const FIELD_USER_ATTR = '[name=user_attr]';
+export const FIELD_USER_ATTR_VALUE = 'User attribute value';
+export const FIELD_USER_FILTER = '[name=user_filter]';
+export const FIELD_USER_FILTER_VALUE = 'User filter value';
+export const FIELD_ACCOUNT_ATTRIBUTE_MAPS_FROM =
+  '[name=account_attribute_maps] input';
+export const FIELD_ACCOUNT_ATTRIBUTE_MAPS_FROM_VALUE = 'attribute';
+export const FIELD_ACCOUNT_ATTRIBUTE_MAPS_TO =
+  '[name=account_attribute_maps] select';
+export const FIELD_ACCOUNT_ATTRIBUTE_MAPS_TO_VALUE = 'email';
+export const FIELD_ACCOUNT_ATTRIBUTE_MAPS_ADD_BTN =
+  '[name=account_attribute_maps] button';
+export const FIELD_GROUP_DN = '[name=group_dn]';
+export const FIELD_GROUP_DN_VALUE = 'Group dn value';
+export const FIELD_GROUP_ATTR = '[name=group_attr]';
+export const FIELD_GROUP_ATTR_VALUE = 'Group attribute value';
+export const FIELD_GROUP_FILTER = '[name=group_filter]';
+export const FIELD_GROUP_FILTER_VALUE = 'Group filter value';
+export const FIELD_ENABLE_GROUPS = '[name=enable_groups]';
+export const FIELD_USE_TOKEN_GROUPS = '[name=use_token_groups]';
+export const FIELD_ERROR_ = '.hds-form-error__message'; // Selects any error message on the DOM.
```

Comment:
this selector has an extra underscore at the end, im wondering if it should just be `FIELD_ERROR` instead?

Reply:
That's a nice catch, sharp eyes Lis!! send a fix 😉 thanks for the catch